pkg/backend/identityapi/register_client.go

```diff
@@ -42,10 +53,13 @@ type RegisterEntity struct {
 }
 
 type RegisterEntityResponse struct {
-	ID  entity.ID  `json:"entityID"`
-	Key entity.Key `json:"entityKey"`
+	ID   entity.ID  `json:"entityID"`
+	Key  entity.Key `json:"entityKey"`
+	Name string     `json:"entityName"`
 }
 
+type RegisterBatchEntityResponse []RegisterEntityResponse
+
 func NewRegisterEntity(key entity.Key) RegisterEntity {
 	return RegisterEntity{key, "", "", nil, nil}
```

Comment:
I guess we are able to register name, tyep & tags now, should this be commented as TODO or added into a to-do list for the happy path?

Reply:
This part of the old workflow/vortext part of the code. If the plan is to update that path as well then we should add it as a to-do list.